File: app/src/main/java/mb00/android/codehub/ui/gist/inject/GistCommentsFragmentComponent.java
package mb00.android.codehub.ui.gist.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.gist.view.GistCommentsFragment;


@Subcomponent(modules = GistCommentsFragmentModule.class)
public interface GistCommentsFragmentComponent extends AndroidInjector<GistCommentsFragment> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<GistCommentsFragment> {}

}

File: app/src/main/java/mb00/android/codehub/ui/gist/inject/GistCommentsFragmentComponent.kt
package mb00.android.codehub.ui.gist.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.gist.view.GistCommentsFragment


@Subcomponent(modules = [
    GistCommentsFragmentModule::class
])
interface GistCommentsFragmentComponent : AndroidInjector<GistCommentsFragment> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<GistCommentsFragment>()

}
